Code/GraphMol/Substruct/SubstructMatch.cpp

```diff
@@ -28,16 +29,17 @@ namespace RDKit{
     typedef std::map<unsigned int,QueryAtom::QUERYATOM_QUERY *> SUBQUERY_MAP;
     
     void MatchSubqueries(const ROMol &mol,QueryAtom::QUERYATOM_QUERY *q,bool useChirality,
-			 SUBQUERY_MAP &subqueryMap,bool useQueryQueryMatches);
+			 SUBQUERY_MAP &subqueryMap,bool useQueryQueryMatches
+#ifdef RDK_THREADSAFE_SSS
```

Comment:
I am not really comfortable with the function arguments changing based on compile flags.
If `RDK_THREADSAFE_SSS` has not been defined, then the vector will always be empty. Seems like there's no harm in having the argument always there.